Core/EntityManager/src/test/java/eu/neclab/ngsildbroker/entityhandler/services/EntityServiceTest.java

```diff
@@ -1,298 +1,374 @@
+					"http://example.org/vehicle/speed", resolved);
+			Assert.assertEquals(updateResult.getUpdated(), request.getUpdateResult().getUpdated());
+		} catch (Exception ex) {
+			Assert.fail();
+		}
+	}
+
+	// TODO redo
+	@Test
+	public void updatePartialDefaultAttributeFieldTest() {
+		try {
+			ArrayListMultimap<String, String> entityIds = ArrayListMultimap.create();
+			entityIds.put(AppConstants.INTERNAL_NULL_KEY, "urn:ngsi-ld:Vehicle:A103");
+			when(entityInfoDAO.getAllIds()).thenReturn(entityIds);
+			Method postConstruct = EntityService.class.getDeclaredMethod("loadStoredEntitiesDetails");
+			postConstruct.setAccessible(true);
+			postConstruct.invoke(entityService);
+			ReflectionTestUtils.setField(entityService, "directDB", true);
+			UpdateResult updateResult = new UpdateResult();
+			Mockito.doReturn(entityPayload).when(entityInfoDAO).getEntity(any(), any());
+			multimaparr.put("content-type", "application/json");
+			Gson gson = new Gson();
+			Map<String, Object> resolved = gson.fromJson(updatePartialDefaultAttributesPayload, Map.class);
+			Map<String, Object> entityBody = (Map<String, Object>) JsonUtils.fromString(entityPayload);
+			UpdateEntityRequest request = new UpdateEntityRequest(multimaparr, "urn:ngsi-ld:Vehicle:A103", entityBody,
+					resolved, "http://example.org/vehicle/speed");
+			updateResult = entityService.partialUpdateEntity(multimaparr, "urn:ngsi-ld:Vehicle:A103",
+					"http://example.org/vehicle/speed", resolved);
+			Assert.assertEquals(updateResult.getUpdated(), request.getUpdateResult().getUpdated());
+		} catch (Exception ex) {
+			Assert.fail();
+		}
+	}
+
+	/**
 	 * this method is use for the datasetId is exist in case of delete the attribute
 	 * instance
 	 */
-/*
- * //TODO redo
- * 
- * @Test public void deleteAttributeInstanceIfDatasetIdExistTest() { try {
- * Mockito.doReturn(payloadNode).when(objectMapper).readTree(any(String.class));
- * entityService.deleteFields(entityPayload, "http://example.org/vehicle/speed",
- * "urn:ngsi-ld:Property:speedometerA4567-speed",null); } catch (Exception ex) {
- * Assert.fail(); } }
- * 
- * 
- *//**
+	// TODO redo
+
+	@Test
+	public void deleteAttributeInstanceIfDatasetIdExistTest() {
+		try {
+			ArrayListMultimap<String, String> entityIds = ArrayListMultimap.create();
+			entityIds.put(AppConstants.INTERNAL_NULL_KEY, "urn:ngsi-ld:Vehicle:A103");
+			when(entityInfoDAO.getAllIds()).thenReturn(entityIds);
+			Method postConstruct = EntityService.class.getDeclaredMethod("loadStoredEntitiesDetails");
+			postConstruct.setAccessible(true);
+			postConstruct.invoke(entityService);
+			ReflectionTestUtils.setField(entityService, "directDB", true);
+			Mockito.doReturn(entityPayload).when(entityInfoDAO).getEntity(any(), any());
+			multimaparr.put("content-type", "application/json");
+			entityService.deleteAttribute(multimaparr, "urn:ngsi-ld:Vehicle:A103", "http://example.org/vehicle/speed",
+					"urn:ngsi-ld:Property:speedometerA4567-speed", "deleteAll");
```

Comment:
[Major] this test case is not complete also due to not compare the desire result.